eraser-map.yaml

```diff
@@ -1466,7 +1462,7 @@ layers:
                         width: [[9, 0px], [10, 0.5px], [14, 2px], [16, 4m], [17, 5m], [18, 6m]]
                         outline:
                             color: [[9, [0.827,0.827,0.827]], [13, [0.827,0.827,0.827]], [15, [1.000,0.739,0.710]]]
-                            width: [[9, 0px], [10, 1px], [14, 1px], [16, 1.5px], [17, 2px]]
+                            width: [[13, 1px], [14, 1px], [16, 1.5px], [17, 2px]]
```

Comment:
In theory it is, in the screenshots, I'm having a hard time identifying these changes except as a handful of pixels (maybe I have more tolerance). The performance impact of theses changes is real, -12% at z9 but still -7% at z11. So if you see specific issues I think we can likely adjust the style around it, but as is it just did not seem worth adding the weight to the scene for what practically amounts to noise IMO :)  

Sub-pixel outlines can also cause aliasing artifacts so I think it's also worth trying out alternatives, in some cases it may look better without (this is subjective though). I particularly want to avoid sub-pixel outlines at z8-10 where possible. Even when the width is 0px at a particular zoom, the line is built and rendered at that zoom if the *next* zoom has a width, because it's interpolating to the next zoom's value.

If we need to go earlier here perhaps we could start the outline at z12 instead of z13 (which means it would build lines at z11), or is there a bigger visual impact I'm not seeing?